setup.py

```diff
@@ -34,7 +34,8 @@ def _get_version():
     ["ml-pipelines-sdk>=0.26.3<2.0.0", "ml-metadata>=0.26<2.0.0"],
     "schema_curation": [
         "tfx>=0.26.3<2.0.0",
-    ]
+    ],
+    "xgboost_evaluator": ["xgboost>=1.4.0"]
```

Comment:
#57 required 1.0.0. Should they all line up?
https://github.com/tensorflow/tfx-addons/pull/57/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R38

Reply:
If there is no error when setting this to `>=1.0.0`, I'd say let's use that. Major version 1 is supposed to have some level of backward compatibility.

Reply:
Set to >=1.0.0.